analysis_options.yaml

```diff
@@ -19,7 +19,6 @@ linter:
     avoid_void_async: true
     cascade_invocations: true
     directives_ordering: true
-    invalid_annotation_target: false
```

Comment:
Got this error in VSCode, nothing in the console, so I'm not 100% if it's accurate.

`'invalid_annotation_target' is not a recognized lint ruledart(undefined_lint_warning)`